Makefile

```diff
@@ -148,12 +148,13 @@ initramfs: ## Builds the compressed initramfs and outputs it to the artifact dir
 
 .PHONY: installer
 installer: ## Builds the container image for the installer and outputs it to the artifact directory.
-ifeq (,$(MULTI_PLATFORM))
-	@$(MAKE) docker-$@ DEST=$(ARTIFACTS) TARGET_ARGS="--allow security.insecure"
-	@docker load < $(ARTIFACTS)/$@.tar
-else
 	@$(MAKE) registry-$@ TARGET_ARGS="--allow security.insecure"
-endif
+# ifeq (,$(MULTI_PLATFORM))
```

Comment:
we could drop "old way" via `docker load` if we want to

Reply:
Not sure I understand, but to explain why I commented this out ... it seems targeting a specific platform for the installer is either broken or I am not doing it right.

Reply:
Whoops I have to revert this change.